shared/src/commonMain/kotlin/io.newm.shared/public/usecases/WalletNFTTracksUseCase.kt

```diff
@@ -24,6 +26,8 @@ interface WalletNFTTracksUseCase {
      * @return Flow<List<NFTTrack>> - A flow emitting the list of NFT tracks associated with the wallet.
      */
     fun getAllNFTTracksFlow(): Flow<List<NFTTrack>>
+
+    @Throws(KMMException::class, CancellationException::class)
```

Comment:
Why the we remove this annotations from the API level?

Reply:
only needed at ios-kmm boundary